ftw/simplelayout/locales/de/LC_MESSAGES/ftw.simplelayout.po

```diff
@@ -556,8 +575,38 @@ msgstr "Datei hochladen"
 msgid "label_video_url"
 msgstr "Video-URL"
 
+#. Default: "height: ${height}px (current: ${current_height}px)"
+#: ./ftw/simplelayout/images/validators.py:63
+msgid "limit_height_str"
+msgstr "Höhe: ${height}px (aktuell: ${current_height}px)"
+
+#. Default: "${width_str} and ${height_str}"
+#: ./ftw/simplelayout/images/validators.py:79
+msgid "limit_width_and_height"
+msgstr "${width_str} und ${height_str}"
+
+#. Default: "width: ${width}px (current: ${current_width}px)"
+#: ./ftw/simplelayout/images/validators.py:57
+msgid "limit_width_str"
+msgstr "Breite: ${width}px (aktuell: ${current_width}px)"
+
+#. Default: "low image quality"
+#: ./ftw/simplelayout/images/templates/macros.pt:5
+msgid "low_quality_image_label"
+msgstr "Bildauflösung ist nicht optimal"
+
+#. Default: "The image is not in optimal resolution. Make sure, your image has at least the following dimensions: ${limit_str}"
+#: ./ftw/simplelayout/images/validators.py:44
+msgid "soft_limit_not_satisfied"
+msgstr "Das Bild ist nicht in einer optimalen Auflösung vorhanden. Stelle sicher, dass das Bild mindestens in folgender Auflsösung vorhanden ist: ${limit_str}"
```

Comment:
In Standardmodule verwenden wir normalerweise die Sie-Form. Ich würde das aus Konsistenzgründen hier auch machen.